src/bit_array.cr

```diff
@@ -125,4 +222,20 @@ struct BitArray
   private def malloc_size
     (@size / 32.0).ceil.to_i
   end
+
+  # FIXME: this was copied from Array, we should deduplicate implementations
```

Comment:
We could put this method as a nodoc method on `Enumerable` or `Indexable`. But if we find its ugly to deduplicate then yes, duplicating the message would be wise.